src/platform/Kconfig

```diff
@@ -305,14 +305,14 @@ config CAVS_LPRO_ONLY
 	  This option is for debugging only at the moment, choose n if unclear.
 
 config CAVS_USE_LPRO_IN_WAITI
-	bool
+	bool "Use low power ring oscillator in WFI"
 	default n
-	depends on CAVS
+	depends on CAVS && !CAVS_LPRO_ONLY
 	select WAKEUP_HOOK
 	help
-	  Select if platform requires DSP clock source to be switched to LPRO
-	  while in waiti.
-	  After waiti clock source is restored.
+	  Select if we want to use LPRO clock in waiti.
```

Comment:
... in order to save power.

Reply:
It was developed not for saving power in the first place, it's just requirement from HW, otherwise you can get hang, cos pre2.x cavs is not supporting hpro in waiti.